Resources/Prototypes/_Goobstation/Catalog/Fills/Belt/belts.yml

```
@@ -1,5 +1,6 @@
 - type: entity
-  parent: ClothingBeltMilitaryWebbingCMO
+  parent: [ ClothingBeltMilitaryWebbingCMO, BaseMedicalContraband ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)